[2019-10-17 01:24:06,205] {taskinstance.py:616} INFO - Dependencies all met for <TaskInstance: dag_cluster.Begin_execution 2018-01-03T00:00:00+00:00 [queued]>
[2019-10-17 01:24:06,218] {taskinstance.py:616} INFO - Dependencies all met for <TaskInstance: dag_cluster.Begin_execution 2018-01-03T00:00:00+00:00 [queued]>
[2019-10-17 01:24:06,219] {taskinstance.py:834} INFO - 
--------------------------------------------------------------------------------
[2019-10-17 01:24:06,220] {taskinstance.py:835} INFO - Starting attempt 1 of 1
[2019-10-17 01:24:06,221] {taskinstance.py:836} INFO - 
--------------------------------------------------------------------------------
[2019-10-17 01:24:06,289] {taskinstance.py:855} INFO - Executing <Task(DummyOperator): Begin_execution> on 2018-01-03T00:00:00+00:00
[2019-10-17 01:24:06,290] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'dag_cluster', 'Begin_execution', '2018-01-03T00:00:00+00:00', '--job_id', '5', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/cluster_dag.py', '--cfg_path', '/tmp/tmp65a11i9x']
[2019-10-17 01:24:07,818] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution [2019-10-17 01:24:07,815] {plugins_manager.py:148} ERROR - No module named 'operators'
[2019-10-17 01:24:07,819] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution Traceback (most recent call last):
[2019-10-17 01:24:07,819] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "/usr/local/lib/python3.7/site-packages/airflow/plugins_manager.py", line 142, in <module>
[2019-10-17 01:24:07,820] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution     m = imp.load_source(namespace, filepath)
[2019-10-17 01:24:07,820] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "/usr/local/lib/python3.7/imp.py", line 171, in load_source
[2019-10-17 01:24:07,820] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution     module = _load(spec)
[2019-10-17 01:24:07,821] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 696, in _load
[2019-10-17 01:24:07,821] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
[2019-10-17 01:24:07,822] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
[2019-10-17 01:24:07,824] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
[2019-10-17 01:24:07,824] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "/usr/local/airflow/plugins/__init__.py", line 3, in <module>
[2019-10-17 01:24:07,825] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution     import operators
[2019-10-17 01:24:07,827] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution ModuleNotFoundError: No module named 'operators'
[2019-10-17 01:24:07,828] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution [2019-10-17 01:24:07,819] {plugins_manager.py:149} ERROR - Failed to import plugin /usr/local/airflow/plugins/__init__.py
[2019-10-17 01:24:07,869] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution [2019-10-17 01:24:07,863] {plugins_manager.py:148} ERROR - No module named 'botocore'
[2019-10-17 01:24:07,869] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution Traceback (most recent call last):
[2019-10-17 01:24:07,870] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "/usr/local/lib/python3.7/site-packages/airflow/plugins_manager.py", line 142, in <module>
[2019-10-17 01:24:07,870] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution     m = imp.load_source(namespace, filepath)
[2019-10-17 01:24:07,875] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "/usr/local/lib/python3.7/imp.py", line 171, in load_source
[2019-10-17 01:24:07,878] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution     module = _load(spec)
[2019-10-17 01:24:07,880] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 696, in _load
[2019-10-17 01:24:07,881] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
[2019-10-17 01:24:07,881] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
[2019-10-17 01:24:07,882] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
[2019-10-17 01:24:07,883] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "/usr/local/airflow/plugins/custom_operator.py", line 8, in <module>
[2019-10-17 01:24:07,894] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution     from airflow.hooks.S3_hook import S3Hook
[2019-10-17 01:24:07,896] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/S3_hook.py", line 19, in <module>
[2019-10-17 01:24:07,898] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution     from botocore.exceptions import ClientError
[2019-10-17 01:24:07,899] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution ModuleNotFoundError: No module named 'botocore'
[2019-10-17 01:24:07,900] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution [2019-10-17 01:24:07,869] {plugins_manager.py:149} ERROR - Failed to import plugin /usr/local/airflow/plugins/custom_operator.py
[2019-10-17 01:24:07,910] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution [2019-10-17 01:24:07,909] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-10-17 01:24:08,501] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution [2019-10-17 01:24:08,500] {dagbag.py:90} INFO - Filling up the DagBag from /usr/local/airflow/dags/cluster_dag.py
[2019-10-17 01:24:08,658] {base_task_runner.py:115} INFO - Job 5: Subtask Begin_execution [2019-10-17 01:24:08,657] {cli.py:516} INFO - Running <TaskInstance: dag_cluster.Begin_execution 2018-01-03T00:00:00+00:00 [running]> on host 614b7d808020
[2019-10-17 01:24:11,167] {logging_mixin.py:95} INFO - [[34m2019-10-17 01:24:11,167[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 0[0m
